app/assets/stylesheets/blacklight/_controls.scss

```diff
@@ -0,0 +1,61 @@
+#sortAndPerPage, .pagination-search-widgets {
+  border-bottom: 1px solid $pagination-border;
+  margin-bottom: 1em;
+  padding-bottom: 1em;
+}
+
+.pagination-search-widgets {
+  @extend .clearfix;
+  padding-top: 1px;
+  padding-bottom: $padding-base-vertical;
+}
+
+#sortAndPerPage .dropdown-toggle {
+  cursor: pointer;
+}
+
+.no-js #sortAndPerPage {
+  & {
+    @extend .clearfix;
+  }
+  .dropdown-menu {
+    background: none;
+    box-shadow: none;
```

Comment:
Properties should be ordered background, border, box-shadow, display, float, position